src/pages/movieDetailing.js

```diff
@@ -4,13 +4,13 @@ import {renderPageLayout} from '../layouts/createLayout.js';
 
 export function createMovieInfo() {
     const movieDetailBlock = createStructurePageMovieDetailing();
- 
-    renderPageLayout(movieDetailBlock);
+    const mainClassName = 'main--movie-info';
```

Comment:
@Mmarvinn Why do you need, here new variables? Pls pass this className direct to the function
```js
renderPageLayout(movieDetailBlock, 'main--movie-info');
```

Reply:
fix